python/PiFinder/ui/object_list.py

```diff
@@ -236,6 +260,16 @@ def _obj_to_mag_color(self, obj: CompositeObject) -> int:
         """
         return self._interpolate_color(self._safe_obj_mag(obj))
 
+    def _get_scrollspeed_config(self):
+        scroll_dict = {
+            "Off": 0,
+            "Fast": TextLayouterScroll.FAST,
+            "Med": TextLayouterScroll.MEDIUM,
+            "Slow": TextLayouterScroll.SLOW,
+        }
+        scrollspeed = self._config_options["Scrolling"]["value"]
```

Comment:
None of the UI modules will have _config_options any longer.  This can/should go into the main config file via the config object `self.config_object` which is a shared instance of the `Config` class which has these methods:

```python
    def set_option(self, option, value):
        self._config_dict[option] = value
        self.dump_config()

    def get_option(self, option, default: Any = None):
        return self._config_dict.get(
            option, self._default_config_dict.get(option, default)
        )
```

To allow people to configure this, we'll add a sub-menu to the menu configuration dictionary under 'Settings'... which I still need to finish porting some previous options to and probably add some more hierarchy to like a Display sub-menu!

Reply:
Resolving as this is on my Todo list now 🤝